tools/ci_connector_ops/ci_connector_ops/pipelines/contexts.py

```diff
@@ -43,20 +130,20 @@ def __init__(
         pipeline_start_timestamp: Optional[int] = None,
         ci_context: Optional[str] = None,
     ):
+        pipeline_name = f"CI test for {connector.technical_name}"
+        super().__init__(
```

Comment:
As the parent init is responsible for updating commit status check can we call super at the end of the child context constructor?